dotnet/src/webdriver/BiDi/Communication/Broker.cs

```diff
@@ -64,6 +64,7 @@ internal Broker(BiDi bidi, ITransport transport)
             PropertyNameCaseInsensitive = true,
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase,
             DefaultIgnoreCondition = JsonIgnoreCondition.WhenWritingNull,
+            NumberHandling = JsonNumberHandling.AllowNamedFloatingPointLiterals,
```

Comment:
In Firefox:
```json
{"type":"number","value":"-0"}
```

Number in quotes.

```suggestion
            NumberHandling = JsonNumberHandling.AllowNamedFloatingPointLiterals | JsonNumberHandling.AllowReadingFromString,
```